2022-04-21 14:52:05,541 INFO    MainThread:287363 [wandb_setup.py:_flush():71] setting env: {}
2022-04-21 14:52:05,542 INFO    MainThread:287363 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-21 14:52:05,542 INFO    MainThread:287363 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-21 14:52:05,542 INFO    MainThread:287363 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220421_145205-3ebwa11t/logs/debug.log
2022-04-21 14:52:05,542 INFO    MainThread:287363 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220421_145205-3ebwa11t/logs/debug-internal.log
2022-04-21 14:52:05,543 INFO    MainThread:287363 [wandb_init.py:init():404] calling init triggers
2022-04-21 14:52:05,543 INFO    MainThread:287363 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-21 14:52:05,543 INFO    MainThread:287363 [wandb_init.py:init():460] starting backend
2022-04-21 14:52:05,543 INFO    MainThread:287363 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-21 14:52:05,556 INFO    MainThread:287363 [backend.py:ensure_launched():216] starting backend process...
2022-04-21 14:52:05,561 INFO    MainThread:287363 [backend.py:ensure_launched():221] started backend process with pid: 287436
2022-04-21 14:52:05,562 INFO    MainThread:287363 [wandb_init.py:init():469] backend started and connected
2022-04-21 14:52:05,565 INFO    MainThread:287363 [wandb_init.py:init():533] updated telemetry
2022-04-21 14:52:05,572 INFO    MainThread:287363 [wandb_init.py:init():563] communicating current version
2022-04-21 14:52:05,911 INFO    MainThread:287363 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-21 14:52:05,911 INFO    MainThread:287363 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-21 14:52:06,083 INFO    MainThread:287363 [wandb_init.py:init():606] starting run threads in backend
2022-04-21 14:52:06,153 INFO    MainThread:287363 [wandb_run.py:_console_start():1810] atexit reg
2022-04-21 14:52:06,154 INFO    MainThread:287363 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-04-21 14:52:06,154 INFO    MainThread:287363 [wandb_run.py:_redirect():1689] Redirecting console.
2022-04-21 14:52:06,155 INFO    MainThread:287363 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-21 14:52:06,155 INFO    MainThread:287363 [wandb_init.py:init():633] run started, returning control to user process
2022-04-21 14:52:09,171 INFO    MainThread:287363 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-04-21 14:52:09,172 INFO    MainThread:287363 [wandb_run.py:_restore():1752] restore
2022-04-21 14:52:11,693 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 950
}

2022-04-21 14:52:11,804 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 950
}

2022-04-21 14:52:12,002 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 950
}

2022-04-21 14:52:12,103 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 9023
}

2022-04-21 14:52:12,206 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 950
  total_bytes: 9023
}

2022-04-21 14:52:12,308 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:12,411 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:12,515 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:12,616 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:12,719 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:12,823 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:13,096 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}

2022-04-21 14:52:13,486 INFO    MainThread:287363 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9023
  total_bytes: 9023
}
local_info {
}

2022-04-21 14:52:14,538 INFO    MainThread:287363 [wandb_run.py:_append_files():2180] logging synced files
